.github/actions/install-winget/action.yaml

```diff
@@ -75,7 +76,3 @@ runs:
           Start-Sleep -Seconds 1
         }
         Write-Output "Winget Version: $(winget --version)"
-
```

Comment:
nit:  I'm not sure why this step was removed from here and added to each workflow instead.